### PR TITLE
Adding all Xperia X, XA2 and 10 to Recovery Mode doc

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by jovirkku** [comment]
I have added Xperia X, XA2 & 10 to the same Recovery Mode doc with Xperia 10 II & III. For this:

a) I have modified the phone preparation sections:
Setting up the Recovery Mode on the phone
- Preparing Xperia 10 II or Xperia 10 III to the Recovery Mode
- Preparing Xperia X, Xperia XA2 or Xperia 10 to the Recovery Mode]
- Forcing the phone to the Recovery Mode

b) I have modified the "back to normal" section:
Reverting the phone back to the normal state
- Reverting Xperia 10 II or Xperia 10 III to the normal mode
- Reverting Xperia X, Xperia XA2 or Xperia 10 to the normal mode

c) Due to different chapter names, modified zillion cross-references inside the doc.

**Review by jpwalden — APPROVED** [review]
Very good update, now this article covers other Xperia devices as well.

File: Support/Help_Articles/Recovery_Mode/README.md [hunk 1]
```diff
@@ -71,9 +70,63 @@ sudo fastboot reboot
...
+    If not, install _fastboot_ now - the instructions are:
+    * Windows computer: chapter 2 in **[Sailfish X instructions - Windows](https://jolla.com/sailfishx-windows-instructions/)**
+    * Linux computer: chapter 2 in **[Sailfish X instructions - Linux](https://jolla.com/sailfishx-linux-instructions/)**
+    * Mac computer: chapter 2 in **[Sailfish X instructions - Mac](https://jolla.com/sailfishx-macos-instructions/)**
+4. Force your Xperia to the _fastboot mode_:
+    * Hold the Volume Up key down and connect the data-connected USB cable
+    * Ensure that the LED at the top of the display area is lit in BLUE colour
+      _(if not, disconnect the cable, release the key and try again)_
+5. You will need the same version of `hybris-recovery.img` as belongs to the Sailfish OS version of your phone currently. In other words, if you have updated OS version 4.3.0 (say) to your phone, then you should use the hybris-recovery.img of OS version 4.3.0. To ensure this, download the latest Sailfish OS image from **https://shop.jolla.com/downloads/** (provided that you have updated your phone to the latest).  Unzip the archive and observe the hybris-recovery.img file in it.
```
Comment (vigejolla) [inline Support/Help_Articles/Recovery_Mode/README.md:88]: Maybe `[**Jolla Shop**](https://shop.jolla.com/downloads/)` ?

Comment (jovirkku) [reply Support/Help_Articles/Recovery_Mode/README.md]: Yes, definitely.

**Comment by jovirkku** [comment]
Done.

**Review by vigejolla — APPROVED** [review]

**Pull request merged**